getting-started-github/README.md

```diff
@@ -0,0 +1,37 @@
+
+Programmers use Git to store multiple versions of their code (hence the name Version Control System (VSC)). Github is the most popular platform to host these revisions. In fact, you're looking at this code on a github repoitory right now! Github will allow you to share your code, collaborate with other developers and test certain aspects of your codebase! There are competitors to Github out there (like Bitbucket), but since I haven't used them, I won't comment on their functionality/usefulness. 
+
+## How is github organized?
+Your code lives in this object called a **repository** (repo). Within that repository, you can have branches, issues, pull requests and a LOT more. Don't get overwhelmed though, for a hackathon you don't need to use all these features. Just follow along with this guide, and we'll have you and your team up and running in no time. 
+
+## How do I set it up on my computer?
+Quick side note: [This official tutorial](https://help.github.com/en/github/getting-started-with-github/set-up-git) is pretty good at explaining how to use certain features of Github. However, I didn't really think it was very useful when I was a beginner. I'll do my best to break down essential steps in this guide, but refer to the tutorial to see specific buttons and stuff you have to click on. 
+
+First of all, register for an account on the [website](github.com). Next, if you are a new to git, I'd recommend downloading the desktop version of github just to help you get everything on a GUI (Graphical User Interface). As you get more advanced, you might find yourself ditching the GUI all together in favor of the terminal/command line, but we don't need to worry about that yet. 
+
+Once you have Github Desktop installed, we can get started. Create a repository on the website (github.com) when you're logged in. Once you have created it, you should **clone** it to your computer using the github desktop program. **Clone** means to copy the code exactly as it is on the github server to your local computer. Once you clone it, you should be able to navigate back to it on your computer. 
+
+Go ahead and add a file. It doesn't have to be code, Github supports uploading almost any type of file (.pdf,.py, .o, etc.). Once you have added a file to the repository folder, navigate back to Github Desktop. You need to **commit** your changes. Ideally, you want to commit ever so often when you are working on a project. Think of it like hitting "save" on a word document. That's not what really happens behind the scenes at all, but its a pretty basic analogy that lets you get past this initial hurdle. You will also have to enter a commit message on the desktop app. This isn't always required, but usually it helps your teammates know what changes you made. This might also come in handy if (when?) you accidentally save a bug, and need to **revert** back to an older version. Check out the "Where can I learn more?" section for a couple resouces for this.
```

Comment:
```suggestion
Go ahead and add a file. It doesn't have to be code, Github supports uploading almost any type of file (.pdf, .py, .o, etc.). Once you have added a file to the repository folder, navigate back to Github Desktop. You need to **commit** your changes. Ideally, you want to commit ever so often when you are working on a project. Think of it like hitting "save" on a word document. That's not what really happens behind the scenes at all, but its a pretty basic analogy that lets you get past this initial hurdle. You will also have to enter a commit message on the desktop app. This isn't always required, but usually it helps your teammates know what changes you made. This might also come in handy if (or when) you accidentally save a bug, and need to **revert** back to an older version. Check out the "Where can I learn more?" section for a couple resources for this.
```